.vsts-ci.yml

```diff
@@ -239,7 +270,7 @@ jobs:
           KeyVaultName: EngKeyVault
           SecretsFilter: 'dotnet-myget-org-api-key'
       - script: powershell -ExecutionPolicy ByPass -NoProfile -command "& """$(Build.SourcesDirectory)\eng\common\Build.ps1""" -restore" &&
-          for /f %%x in ('dir /s /b $(Build.SourcesDirectory)\artifacts\packages\pushToMyGet\*.nupkg') do $(Build.SourcesDirectory)\.dotnet\dotnet.exe nuget push %%x -k $(dotnet-myget-org-api-key) -s https://dotnet.myget.org/F/dotnet-core/
+          for /f %%x in ('dir /s /b $(Build.SourcesDirectory)\artifacts\packages\pushToMyGet\*.nupkg') do dotnet nuget push %%x -k $(dotnet-myget-org-api-key) -s https://dotnet.myget.org/F/dotnet-core/
```

Comment:
Does Arcade really not provide support for publishing to myget?

Rather than using the `AzureKeyVault` task above, I'd suggest you use the "DotNet-MyGet-Publish" variable group.

```yaml
variables:
  - group: DotNet-MyGet-Publish
```

Reply:
It might support it, but at least it didn't back when I initially added this since we were moving away from it. I'll switch to use the group as you suggest.